code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -46,6 +49,10 @@
 			filter_types -= to_filter
 			filter_types += gas_id2path(to_filter)
 
+/obj/machinery/atmospherics/components/unary/vent_scrubber/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/atmos_sensitive, mapload)
```

Comment:
Keep in mind that scrubbers can also take from the surrounding tiles so this may end up with cases where the scrubber could take gas from nearby tiles but won't because it doesn't wake up.

Reply:
shouldn't be too much of an issue tho, gases moves pretty often so having one tile without a specific gas is very rare

Reply:
yes i expect that to matter for approximately 1-5 atmos ticks

Reply:
does expanded range just go faster?

Reply:
no it makes it scrub all open turfs directly adjacent to the scrubber as well as the turf the scrubber is on